RELEASE_NOTES.rst

```diff
@@ -1,3 +1,13 @@
+coala-bears 0.8.4
+=================
+
+This bugfix release fixes the following issues:
```

Comment:
youre saying: "the following issues" but what youre listing is things that are now working well.